dashboard/app/models/levels/level.rb

```diff
@@ -97,6 +97,7 @@ class Level < ApplicationRecord
     ai_tutor_available
     offer_browser_tts
     use_secondary_finish_button
+    enable_micro_bit
```

Comment:
Should this property just be defined on `pythonlab.rb`? Will we use it elsewhere?

Reply:
Oh yeah - I don't believe this would be used in other labs since it is python-language specific so will move to `pythonlab.rb`. Thanks!